code/datums/mind.dm

```diff
@@ -657,28 +660,32 @@
 /datum/mind/proc/take_uplink()
 	qdel(find_syndicate_uplink())
 
-/datum/mind/proc/make_Traitor()
+/datum/mind/proc/make_traitor()
 	if(!(has_antag_datum(/datum/antagonist/traitor)))
 		add_antag_datum(/datum/antagonist/traitor)
 
-/datum/mind/proc/make_Contractor_Support()
+/datum/mind/proc/make_malfunctioning()
```

Comment:
This whole proc is unnecessary, these were made before antag datums were settled I think. `add_antag_datum` is a no-op if they already have the antag datum.

Reply:
Removed